src/main/java/core/basesyntax/service/impl/ParserOperationServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserOperationService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParserOperationServiceImpl implements ParserOperationService {
+    private static final String SEPARATOR_IN_LINE = ",";
+    private static final int INDEX_FOR_OPERATION_IN_STRING = 0;
+    private static final int INDEX_FOR_PRODUCT_NAME_IN_STRING = 1;
+    private static final int INDEX_FOR_PRODUCT_VALUE_IN_STRING = 2;
+
+    @Override
+    public List<FruitTransaction> parser(List<String> data) {
+
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
```

Comment:
Consider using a more descriptive variable name instead of 'fruitTransactionList', such as 'fruitTransactions'.